### PR TITLE
The front-end for a user to enter upload verification code given to them (Addressing issue #31)

### PR DESCRIPTION
**Pull request by JoshuaBuditama**
This PR contains a simple front end for the upload page, where a user can enter token given to them.
The format of the token addressed in this UI is all uppercase alphabets and with maximum length of 4.
The format of the token can be changed later if required.
Should close issue #31

### CODE REVIEW

File: front-end/src/components/UploadForm.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      
+              {/*Submit button*/}
+              <div className="column is-half">
+                <div className="field">
+                  <p className="control">
+                    <input
+                      type="submit"
+                      className="button is-link"
+                      value="I consent to sharing my close contacts"
+                      disabled={(!verificationToken.match(/^[A-Z]+$/) || verificationToken.length !== 4)}
```
Comment (cskeogh) [inline front-end/src/components/UploadForm.tsx]: `verificationToken.match(/^[A-Z]+$/)` code is repeated three times in this file. Place in a function to help maintainability (the verification code may have to change depending on the server).

File: front-end/src/components/UploadForm.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  }
+  
+  return (
+    <div>
+      {/**Header section*/}
+      <section className="section is-medium has-text-centered">
+        <div className="container">
+          <h1 className="title">Sharing my close contacts with healthcare professional</h1>
+          <h2 className="subtitle">
+            Please enter the healthcare code provided to you by the healthcare professional
```
Comment (cskeogh) [inline front-end/src/components/UploadForm.tsx]: [master L12](https://github.com/JoshuaBuditama/slowvid/blob/master/front-end/src/components/Notifications.tsx#L12) uses the term 'health care' (note space)

File: front-end/src/components/UploadForm.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      //display error pop-up message
+    }
+  }
+  
+  return (
+    <div>
+      {/**Header section*/}
+      <section className="section is-medium has-text-centered">
+        <div className="container">
+          <h1 className="title">Sharing my close contacts with healthcare professional</h1>
```
Comment (cskeogh) [inline front-end/src/components/UploadForm.tsx]: [master L12](https://github.com/JoshuaBuditama/slowvid/blob/master/front-end/src/components/Notifications.tsx#L12) uses the term 'health care' (note space)

**Review by cskeogh — CHANGES_REQUESTED** [review]
Nice work. I like the use of the stylesheet.

File: front-end/src/components/UploadForm.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+  function validToken() {
+    const validTokenCombination = /^[A-Z]+$/;
+    const validTokenLength = 4;
+    if(verificationToken.match(validTokenCombination) && verificationToken.length === validTokenLength)
+    {
+      return true;
+    }
+    return false;
+  }
```
Comment (JoshuaBuditama) [inline front-end/src/components/UploadForm.tsx:14]: @cskeogh I have addressed the maintainability issue by creating a new function that checks the validity of the token.

File: front-end/src/components/UploadForm.tsx [hunk 5]
```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+
+function UploadForm(){
+  const [verificationToken, setVerificationToken] = useState("");
+
+  function validToken() {
+    const validTokenCombination = /^[A-Z]+$/;
+    const validTokenLength = 4;
+    if(verificationToken.match(validTokenCombination) && verificationToken.length === validTokenLength)
```
Comment (JoshuaBuditama) [inline front-end/src/components/UploadForm.tsx:9]: @cskeogh I have addressed the maintainability issue by creating a new function that checks the validity of the token and call it whenever I need to check the validity of the token in the HTML element

**Comment by JoshuaBuditama** [comment]
@cskeogh Thanks for the review!
I have fixed the maintainability issue and have changed 'healthcare' to 'health care' to better reflect what's on the  master branch.
Please let me know if there is any other issue!

**Review by cskeogh — APPROVED** [review]
Looks good. Good to merge

**Pull request merged**